components/Layout.js

```diff
@@ -0,0 +1,19 @@
+import Header from "./Header.js";
+import styled from "styled-components";
+import Head from "next/head.js";
```

Comment:
you don't need ".js" here

```suggestion
import Head from "next/head";
```